db/v-2/events/event_scheduler.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="event-delete-execution-1" author="frolov" context="prod">
```

Comment:
what's about context `scheduler`?

Reply:
If the context is non-existent, then the event is not created. But we used to have a Prod context, and after we changed it, the base probably didn’t drop, so the events weren’t deleted.

Reply:
maybe then change `scheduler` to `prod` for existed scripts?

Reply:
how liquibase will work with such change of a context?

Reply:
yeap, DATABASECHANGELOG contains the column `contexts` . So changing context will be a problem